test/controllers/update/update.beneficial.owner.other.controller.spec.ts

```diff
@@ -164,6 +166,20 @@ describe("UPDATE BENEFICIAL OWNER OTHER controller", () => {
       expect(response.status).toEqual(500);
       expect(response.text).toContain(SERVICE_UNAVAILABLE);
     });
+
+    test(`Renders the ${UPDATE_BENEFICIAL_OWNER_OTHER_PAGE} page when relevant_period=true`, async () => {
+      mockGetApplicationData.mockReturnValueOnce({ ...APPLICATION_DATA_UPDATE_BO_MOCK });
+
+      const resp = await request(app).get(UPDATE_BENEFICIAL_OWNER_OTHER_URL + "?relevant-period=true");
```

Comment:
I was thinking maybe we should put `"relevant-period=true"` inside a config constant, then we can reuse wherever we need to